kolibri/core/device/hooks.py

```diff
@@ -19,3 +21,37 @@ def plugin_url(self, plugin_class, url_name):
     @classmethod
     def provision_url(cls):
         return next(hook.url for hook in cls.registered_hooks)
+
+
+@define_hook(only_one_registered=True)
+class GetOSUserHook(KolibriHook):
+    @abstractmethod
+    def get_os_user(self, auth_token):
+        pass
+
+    @classmethod
+    def retrieve_os_user(cls, auth_token):
+        try:
+            hook = next(cls.registered_hooks)
+        except StopIteration:
+            raise NotImplementedError(
+                "Getting the OS user is not supported on this platform"
+            )
+        return hook.get_os_user(auth_token)
+
+
+@define_hook(only_one_registered=True)
+class CheckIsMeteredHook(KolibriHook):
+    @abstractmethod
+    def check_is_metered(self):
+        pass
+
+    @classmethod
+    def is_metered(cls):
```

Comment:
Perhaps it would be more consistent to rename this class method to something like retrieve_is_metered or check_is_metered and the other as get_is_metered?

Reply:
As the abstractmethod is the one that others need to implement, having that be the simplest probably seems best.

So maybe the abstract method should be `is_metered` and this one would be `check_is_metered`?

Reply:
This isn't blocking at all and is definitely nit picky! But Blaine and I were looking over some of the changes together and noticed that the methods consistently used verbs, while the class properties were nouns. So maybe if the abstract method remained `check_is_metered` then the class method could be `retrieve_is_metered_check` or `get_is_metered_check` to match how `retrieve_os_user` uses `get_os_user`?